bin/run_tests.bash

```diff
@@ -1,3 +1,3 @@
 #!/usr/bin/env bash
 
-docker-compose run --rm --entrypoint bash -c "RAILS_ENV='test' bundle exec rspec spec" app
+docker-compose run --rm --entrypoint bash -c "bundle exec rspec spec" app
```

Comment:
```suggestion
docker-compose run --rm --entrypoint 'bash -c "bundle exec rspec spec"' app
```

When I tried to run the script, I had to quote the entire entry point before it worked -- I'm not sure why it used to work.

Otherwise, looks good.

Reply:
Thanks, sorry for not running this script. I rearranged params and not it works. Would you take a look again, please?